calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
Maybe just add a couple (more if needed) of menu items under the "Calendar" top menu.
Here is a mockup:

![2017-08-11 11_35_04-meetings - odoo](https://user-images.githubusercontent.com/1246629/29210038-8020fcc6-7e89-11e7-8558-f88e066bd0cc.png)


I'm not sure it is relevant enough to move the "Settings/Technical/Calendar" options to "Calendar/Configuration".